tests/apps_test.go

```diff
@@ -180,8 +181,11 @@ var _ = Describe("deis apps", func() {
 			Specify("that user can retrieve logs for that app", func() {
 				sess, err := cmd.Start("deis logs -a %s", &user, app.Name)
 				Eventually(sess).Should(SatisfyAll(
-					Say("INFO \\[%s\\]: %s created initial release", app.Name, user.Username),
-					Say("INFO \\[%s\\]: domain %s added", app.Name, app.Name)))
+					Say(`(.+) (deis\[controller\]: INFO config test\-.* updated)`),
+					Say(`(.*) (deis\[controller\]: INFO test\-.* created initial release)`),
+					Say(`(.*) (deis\[controller\]: INFO appsettings test\-.* updated)`),
+					Say(`(.*) (deis\[controller\]: INFO domain test\-.* added)`),
+					Say(`(.*) (deis\[controller\]: INFO build test\-.* created)`)))
```

Comment:
alright so this wont work with the app we deploy. It does not generate any log output so for now I wont worry about adding this